studio/src/pages/RequestorPage/AddRouteToCollection/AddToRouteForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { useCollections } from "@/queries";
+import { useStudioStore } from "../store";
+import { AddToRouteFormItem } from "./AddToRouteFormItem";
+
+type Props = {
+  onSuccess: () => void;
+};
+
+export function AddToRouteForm(props: Props) {
+  const { data: collections, isLoading } = useCollections();
+  const { activeRoute } = useStudioStore("activeRoute");
+
+  // No active route? Then there's nothing to do here
+  if (!activeRoute) {
+    console.warn("No active route");
+    return null;
+  }
+
+  if (isLoading || !collections) {
+    return <div>Loading...</div>;
+  }
+
+  if (collections.length === 0) {
+    return <div>Empty</div>;
```

Comment:
probably for future, but maybe we always initialize a "starter" collection of some sort? 

maybe for now could just put some helper text like "Create a collection to get started" or something